botcommand_bible.go

```diff
@@ -1,33 +1,56 @@
 package main
 
 import (
+	"fmt"
 	"github.com/PuerkitoBio/goquery"
 	"jaytaylor.com/html2text"
 	"net/http"
 	"strings"
 )
 
 type BibleCommand struct {
-
 }
 
 func (b *BibleCommand) Satisfies(context *MessageContext) bool {
 	return strings.HasPrefix(context.Message.Content, "!bible")
 }
 
 func (b *BibleCommand) Exec(context *MessageContext) {
-	resp, _ :=  http.Get("https://dailyverses.net/random-bible-verse")
-	doc, _ := goquery.NewDocumentFromReader(resp.Body)
-	htmlVerse, _ := doc.Find("div.bibleVerse").First().Html()
+	msgLen := len(strings.Split(context.Message.Content, " "))
+	if msgLen > 1 {
+		b.getBibleVerse(context)
+	} else {
+		b.randomBibleVerse(context)
+	}
+}
 
-	txt , _ := html2text.FromString(htmlVerse)
+func (b *BibleCommand) getBibleVerse(context *MessageContext) {
+	msgBook := strings.Split(context.Message.Content, " ")[1]
+	msgChpt := strings.Split(strings.Split(context.Message.Content, " ")[2], ":")[0]
+	msgVrs := strings.Split(strings.Split(context.Message.Content, " ")[2], ":")[1]
```

Comment:
invalid input leads to crashing the bot, you could check if indexes exist before accessing them and eventually output that the desired bible verse was not found. You can let the bot crash by typing

`!bible asdf`